salt/modules/rabbitmq.py

```diff
@@ -111,15 +117,14 @@ def _get_rabbitmq_plugin():
     This works by taking the rabbitmq-server version and looking for where it
     seems to be hidden in /usr/lib.
     '''
-    rabbitmq = __context__.get('rabbitmq-plugins')
+    global RABBITMQ_PLUGINS
 
-    if rabbitmq is None:
+    if RABBITMQ_PLUGINS is None:
         version = __salt__['pkg.version']('rabbitmq-server').split('-')[0]
-        rabbitmq = ('/usr/lib/rabbitmq/lib/rabbitmq_server-{0}'
-                    '/sbin/rabbitmq-plugins').format(version)
-        __context__['rabbitmq-plugins'] = rabbitmq
+        RABBITMQ_PLUGINS = ('/usr/lib/rabbitmq/lib/rabbitmq_server-{0}'
+                            '/sbin/rabbitmq-plugins').format(version)
```

Comment:
I'm not sure this function should be changing the globals here. Granted it just takes what was there before, but should the `RABBITMQ_PLUGINS` and `RABBITMQCTL` just be functions that return the correct values instead? 

I'm pretty sure that's what `_get_rabbitmq_plugin()` was intended to do originally.